ffi-cdecl/freetype2_decl.c

```diff
@@ -2,10 +2,23 @@
 
 #include "ffi-cdecl.h"
 
-#include <freetype/freetype.h>
-#include <freetype/ftsynth.h>
-#include <freetype/ftoutln.h>
-
+#include FT_FREETYPE_H
+#include FT_SYNTHESIS_H
+#include FT_OUTLINE_H
```

Comment:
These #`include FT__xxx` were replaced with `#include <freetype/xxx.h>` in #1214 .
Is that an explicite revert, or you did not see it/forgot it while rebasing?

Reply:
It was officially deprecated in 2.10.3, which is why it was coincidental to #1214 ;).

(It's a remnant of... DOS (!) era short filename FS limitations).

TL;DR: Yes, please use sane "modern" includes ^^.

Reply:
oops, indeed accidental rebase resolution